### ISSUE TITLE
DL3004 - can not use `gosu`, the problem is not explained

### ISSUE THREAD

**Issue by abitrolly** [hadolint/hadolint#966]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

> Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root.

https://github.com/hadolint/hadolint/wiki/DL3004

### Expected behavior

Sufficient explanation of what is the unpredictable behavior when using `apt-get -y install`.

Complete example of installing and using `gosu` without using `sudo`.

### Actual behavior

`gosu` is not available on Ubuntu, and using that leads to error.
```
#0 0.532 /bin/sh: 1: gosu: not found
```

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
(paste your output here)
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**
